tests/stress_tests/scripts/memcheck_upload.py

```diff
@@ -62,14 +63,9 @@ def metadata_from_manifest(manifest):
         logging.error('Product type %s is not supported', product_type_str)
         return {}
     return {
-        'os_name': product_type[2],
-        'os_version': [product_type[3], product_type[4]],
-        'commit_sha': repo_trigger['revision'],
+        'os': "{} {}.{}".format(product_type[2], product_type[3], product_type[4]),
         'commit_date': repo_trigger['commit_time'],
-        'repo_url': repo_trigger['url'],
         'branch': repo_trigger['branch'],
-        'target_branch': repo_trigger['target_branch'] if repo_trigger["target_branch"] else repo_trigger["branch"],
```

Comment:
We need to keep target branch as well

Reply:
Done. Push new format data only to Postgres and preserved old records format for Mongo. Target branch is also kept.